packages/block-editor/src/components/link-control/style.scss

```diff
@@ -269,6 +277,7 @@ $preview-image-height: 140px;
 
 		.components-text {
 			font-size: 0.9em;
```

Comment:
Same here with ems, this computes to a non even number:

<img width="306" alt="Screenshot 2021-06-24 at 15 10 07" src="https://user-images.githubusercontent.com/1204802/123268440-4fc4a080-d4fe-11eb-933e-a27a40ac3f66.png">

I'd just use $default-font-size here as well, for legibility reasons.